src/components/Menus/ThemeSubMenu/useThemeMenuContent.tsx

```diff
@@ -33,7 +36,15 @@ export const useThemeMenuContent = () => {
         [TrackingEventParameter.SwitchedTemplate]: theme,
       },
     });
+    if (!(theme === 'system' || theme === 'light' || theme === 'dark')) {
```

Comment:
better to use an include to for readability

Reply:
Good idea!